quickstarts/about-quickstarts.md

```diff
@@ -1,37 +1,107 @@
+	- Code comments that explain what the code does
 
-If users should see a result after completing a particular step, make sure to include it. This might mean including a screenshot of a dialog or terminal output.
 
-## About the summary section
+## About the "Overview" section
 
-Use this section to summarize what tasks your user completed during the tutorial, and to summarize what your users now know how to do.
+Use this section to provide:
 
-## About the next steps section
+* Short description of your product and its purpose
+* Describe what the user will accomplish in this quickstart
+* Intended audience for this document 
+* Basic knowledge that you expect the user to have before using this quickstart.
+
+Defining the audience right in the beginning will help the users identify if the quickstart is relevant for them or if they should continue searching.
+
+## About "Before You Begin" section
+
+Use this section to mention any requirements/configuration prerequisites that the user must complete before they start the quickstart. 
+For easy understanding, we recommend that you classify the prerequisites into different categories such as software prerequisites, hardware prerequisites, and so on. 
+
+## About the "Installation" section
+
+Use this section to provide:
+* Basic instructions/commands to install your product
+* Alternatively you can provide a link to the Installation guide
+* For common software installation instructions provide links to the upstream docs
+
+> **Note:**
+> Some quickstarts may not require Installation instructions, in such scenarios, you can use this section to provide steps to tell the users 
+> how to sign up, obtain user credentials to log in, and any other authorization/authentication-related information.
+
+## About the "Steps" section
+
+The steps section is where you describe what the user needs to do. How you write your steps will vary depending on your organization's style guide.
+This template breaks down the quickstart into **parts**. One part of the quickstart may focus on completing several related steps. 
+You're welcome to follow this structure or use the step headings on their own.
+
+**Tips for writing steps in a quickstart**
+* Start the heading with a verb and express the step/part headings as a complete thought. For example, you could use the heading **Connect to the VM instance** instead of just **Connect**.   
+* For each step, provide some background information about the task so users know what they're about to do and why.
+* Remember to orient your users when walking them through each step. If they need to open a particular file or dialog to complete the task, provide that information first.
+* Use plain language and define the terminology of any technical term next to it.
+* Include just one action in a step.
+
+> **Note:**
+> For additional tips on writing steps, see [Writing Procedural Steps](writing-tips.md#writing-procedural-steps) from **The Good Docs Project**.
+
```

Comment:
Describing good steps is probably a significant amout of this guide, and is worth dedicating some more focus to. (These comments are applicable across multiple procedural types, and so maybe should be pulled out into a separate section, as you have done in this Note)
* I think all steps should be numbered, in the form {part}.{step}. Reason: It helps  communication about the steps, such as between student and teacher.
* I like the way you have started talking about the different ways that steps can be framed. I suggest expand on that a bit, and give a few examples, such as using both {Part/Step} or just {Steps}.
* I suggest we should provide example language (probably in the template) of key words to use:
  * Press button (action)
  * Observe (something changed)
  * Show a screenshot of what people will see
* In the quickstart, we don't want to provide a perfect guided path. So we ideally we should tell the user exactly what values to use, rather than leaving it up to the user to select from one of many options.

Reply:
Fixed.